justindujardin/angular-rpg
# Update app to support Tiled 1.9.0

Changes:
- Update map files to latest tiled format
- Add Tiled editor project file
- Add custom property types for the various MapFeatureComponents the game has
- Add a Tiled editor shortcut to follow a DoorFeatureComponent / PortalFeatureComponent to their destination by pressing Ctrl+F on the keyboard

The only notable change to the Tiled maps is that `<object>`s used to have a `type` property that we used for indicating what kind of map feature component to instantiate. Now we check the `class` property instead.

**Tiled 1.0**
```xml
<object name="Guard" type="DialogFeatureComponent" x="272" y="80" width="16" height="16">
```

**Tiled 1.9.0**
```xml
<object id="4" name="Guard" class="DialogFeatureComponent" x="272" y="80" width="16" height="16">
```
